src/rules/no-duplicate-load-rules/README.md

```diff
@@ -0,0 +1,266 @@
+# no-duplicate-at-import-rules
+
+Disallow duplicate `@import` and `@use` rules.
+
+<!-- prettier-ignore -->
+```css
+    @import "a.css";
+    @import "a.css";
+/** ↑
+ * These are duplicates */
+```
+
+<!-- prettier-ignore -->
+```scss
+    @use "a";
+    @use "a";
+/** ↑
+ * These are duplicates */
+```
+
+The [`message` secondary option](../../../docs/user-guide/configure.md#message) can accept the arguments of this rule.
+
+## Options
+
+### `true`
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+@import 'a.css';
+@import 'a.css';
+```
+
+<!-- prettier-ignore -->
+```css
+@import url("a.css");
+@import url("a.css");
+```
+
+<!-- prettier-ignore -->
+```css
+@import "a.css";
+@import 'a.css';
+```
+
+<!-- prettier-ignore -->
+```css
+@import "a.css";
+@import 'b.css';
+@import url(a.css);
+```
+
+<!-- prettier-ignore -->
+```scss
+@use "a";
+@use "a";
+```
+
+<!-- prettier-ignore -->
+```scss
+@use "a" as abc;
+@use "a" as xyz;
+```
+
+<!-- prettier-ignore -->
+```scss
+@use 'a' with (
+$black: #222,
+$border-radius: 0.1rem
+);
+@use 'a' with ($blue: #333);
+```
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+@import "a.css";
+@import "b.css";
+```
+
+<!-- prettier-ignore -->
+```css
+@import url('a.css') projection;
+@import url('a.css') tv;
+```
+
+<!-- prettier-ignore -->
+```scss
+@use "a";
+@use "b";
+```
+
+<!-- prettier-ignore -->
+```scss
+@use "a" as abc;
+@use "b" as xyz;
+```
+
+<!-- prettier-ignore -->
+```scss
+@use 'a' with (
+$black: #222,
+$border-radius: 0.1rem
+);
+@use 'b' with ($blue: #333);
+```
+
+## Optional secondary options
+
+### `includeForward: true`
```

Comment:
What's the reason not to enable this by default?

Reply:
After discussing with the team offline, we’ve agreed that `@forward` should be required for the rule, but an error should only be triggered if there are two `@forward` rules with identical configurations, so I’ve removed the option entirely.